modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -73,9 +82,68 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: default
+    ---
+    apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy <6>
+    metadata:
+      name: vs2igw
+      namespace: vs2
+    spec:
+      podSelector:
+        matchLabels:
+          istio: ingressgateway
+      ingress:
+        - {}
+      policyTypes:
+      - Ingress
+    ---
+  apiVersion: autoscaling/v2 <7>
+  kind: HorizontalPodAutoscaler
+  metadata:
+    labels:
+      app: vs2igw
+      istio: ingressgateway
+      release: istio
+    name: vs2igw
+    namespace: vs2
+  spec:
+    maxReplicas: 5
+    metrics:
+    - resource:
+        name: cpu
+        target:
+          averageUtilization: 80
+          type: Utilization
+      type: Resource
+    minReplicas: 2
+    scaleTargetRef:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: vs2igw
+    ---
+    apiVersion: policy/v1
+    kind: PodDisruptionBudget<8>
+    metadata:
+      labels:
+        app: vs2igw
+        istio: ingressgateway
+        release: istio
+      name: gateway
+      namespace: vs2
+    spec:
+      minAvailable: 1 # https://istio.io/latest/docs/setup/additional-setup/gateway/#deploying-a-gateway
+      selector:
+        matchLabels:
+          app: gateway
+          istio: ingressgateway
+    ---
```

Comment:
Please remove 4 leading spaces in lines 123-139.

Reply:
done